### PR TITLE
Require livingdocs-framework via npm package

### PR DESCRIPTION
**Pull request by peyerluk**
## Changelog

- 💎 Update the livingdocs framework from  18.5.0 to 19.4.5

## Internal Changes

- require the framework via a npm package

### CODE REVIEW

File: lib/framework.js [hunk 1]
```diff
@@ -0,0 +1 @@
+module.exports = require('@livingdocs/framework-sdk-prebuild')
```
Comment (marcbachmann) [inline lib/framework.js:1]: Why do you use a custom path?

Comment (peyerluk) [reply lib/framework.js:1]: To hide the 'framework-sdk-prebuild' part a bit. As it's not intended that someone directly requires the framework-prebuild. But it was just a hunch. Not sure if it actually improves anything...

Comment (marcbachmann) [reply lib/framework.js:1]: Exposing `livingdocs-node-sdk/lib/framework` is probably not better 😁 
And people will land here first anyways: https://www.npmjs.com/package/@livingdocs/framework-sdk-prebuild

Comment (peyerluk) [reply lib/framework.js:1]: ...maybe I'll add a README in the prebuild as well stating to use the SDK instead.

**Pull request merged**